semgrep_output_v1.atd

```diff
@@ -495,7 +495,7 @@ type match_intermediate_var <python decorator="dataclass(frozen=True)"> = {
 (* both ecosystem and transitivity below have frozen=True so the generated
  * classes can be hashed and put in sets (see calls to reachable_deps.add()
  * in semgrep SCA code)
- * TODO: use <ocaml repr="classic">, and do the same for manifest
+ * alt: type package_manager
```

Comment:
I would prefer "namespace" over package manager, since e.g. both maven and gradle package managers use the maven ecosystem, since they pull packages from the same place

Reply:
yes but namespace is even more vague.